packages/server/dataloader/customRedisQueries.ts

```diff
@@ -26,17 +27,18 @@ const customRedisQueries = {
     })
   },
   publicTemplates: async (meetingTypes: string[]) => {
-    const r = await getRethink()
-
+    const pg = getKysely()
     const publicTemplatesByType = await Promise.all(
       meetingTypes.map((type) => {
         const templateType = type === 'poker' ? 'poker' : 'retrospective'
         // Will convert to PG by Mar 1, 2023
-        return r
-          .table('MeetingTemplate')
-          .getAll('aGhostTeam', {index: 'teamId'})
-          .filter({isActive: true, type: templateType})
-          .run()
+        return pg
```

Comment:
it sure does! Even better, we can get rid of this caching since PG is so much faster than RethinkDB.
I wanted to make sure this PR was just for refactoring, I'll do an optimization PR soon!